### PR TITLE
Document trigger_services step for SCM/CI Workflow Integration

### PR DESCRIPTION
**Pull request by dmarcoux**
To test this locally:
1. Pull the changes from this PR, then run `daps2docker .`
2. Open the generated user guide in your favorite browser: `firefox /tmp/daps2docker-SXo2tVAc/obs-user-guide/html/book.obs-user/cha.obs.scm_ci_workflow_integration.html` (the string after `daps2-docker-` will be different).

Preview of the changes:
preview-1
preview-2

### CODE REVIEW

**Comment by dmarcoux** [comment]
**Do not merge** before https://github.com/openSUSE/open-build-service/pull/12362 is merged and deployed.

**Comment by dmarcoux** [comment]
[Trigger services with a new workflow step (L)](https://trello.com/c/XJBADVLI/1857-trigger-services-with-a-new-workflow-step-l)

File: xml/obs_scm_ci_workflow_integration.xml [hunk 1]
```diff
@@ -381,11 +381,15 @@ rebuild_master:
           <listitem>
             <para>Set flags for projects, packages, repositories or architectures (set_flags)</para>
           </listitem>
+
+          <listitem>
+            <para>Trgger services of a package (trigger_services)</para>
```
Comment (saraycp) [inline xml/obs_scm_ci_workflow_integration.xml:386]:
```suggestion
            <para>Trigger services of a package (trigger_services)</para>
```

File: xml/obs_scm_ci_workflow_integration.xml [hunk 2]
```diff
@@ -616,6 +620,22 @@ rebuild_master:
...
             architecture: x86_64</screen>
         </sect4>
+
+        <sect4 xml:id="sec.obs.obs_scm_ci_workflow_integration.obs_workflows.steps.trigger_services">
+          <title>Trigger Services of a Package</title>
+
+          <para>Given a project called <emphasis>home:Admin</emphasis> and a
+          package <emphasis>ctris</emphasis>, the step will trigger services of the
+          package <emphasis>home:Admin/ctris</emphasis>.</para>
+
```
Comment (saraycp) [inline xml/obs_scm_ci_workflow_integration.xml:630]: What about adding the following sentence?

"Make sure the `ctris` package contains a correct `_service` file. Refer to [obs-service-tar_scm](https://github.com/openSUSE/obs-service-tar_scm#user-documentation) for more details. "

**Review by saraycp — CHANGES_REQUESTED** [review]

File: xml/obs_scm_ci_workflow_integration.xml (hunk 2 above)
Comment (dmarcoux) [reply xml/obs_scm_ci_workflow_integration.xml:630]: I've pushed a change to mention the `_service` file. Let me know what you think.

**Review by saraycp — APPROVED** [review]

**Comment by krauselukas** [comment]
PR https://github.com/openSUSE/open-build-service/pull/12362 is merged

**Pull request merged**